Report/sections/data_technology/completezza.tex

```diff
@@ -179,13 +184,27 @@ \subsubsection*{Stations}
 \subsection{Miglioramento}
 
 Per il miglioramento della completezza verrà effettuato un completamento di dati incompleti con tecniche specifiche che sfruttano la conoscenza dei dati.\\
-In particolare, alcuni casi vengono facilmente risolti derivando il valore di un specifico attributo da un set di altri attributi, per esempio nel dataset \textbf{Weather} l'attributo \textsc{t\_avg} può essere facilmente calcolato come la media degli attributi \textsc{t\_min} e \textsc{t\_max}.
+\textbf{FIXME questo non l'abbiamo fatto} In particolare, alcuni casi vengono 
```

Comment:
Per questo pensavo che l'aveste fatto voi. In ogni caso andrebbe fatto un lavoro per vedere se i campi null possono essere riempiti.